packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -220,36 +220,49 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         global.accounts.push(...accounts);
       });
 
-      beforeEach(async function () {
-        const basicPlans = await hbarSpendingPlanRepository.findAllActiveBySubscriptionTier(
-          [SubscriptionTier.BASIC],
-          requestDetails,
-        );
-        for (const plan of basicPlans) {
-          await hbarSpendingPlanRepository.delete(plan.id, requestDetails);
-          await ethAddressSpendingPlanRepository.deleteAllByPlanId(plan.id, 'before', requestDetails);
-          await ipSpendingPlanRepository.deleteAllByPlanId(plan.id, 'before', requestDetails);
-        }
```

Comment:
**Note for reviewer:** 
Moved this logc to `BASIC Tier` so it can only be executed within the `BASIC Tier` scope. This change was made as it impacts the tests for `multiple tiered users`.